dashboard/test/ui/features/projects.feature

```diff
@@ -9,8 +9,11 @@ Scenario: Save Artist Project
   And element ".project_updated_at" eventually contains text "Saved"
   Then I open the topmost blockly category "Brushes"
   And I drag block matching selector "#draw-color" to block matching selector "#when_run"
-  Then I click selector ".project_share"
-  And I wait to see "#x-close"
+
+  # cannot publish because not signed in
+  And I open the project share dialog
+  And the project cannot be published
```

Comment:
Nit: This would read better (and be functionally equivalent) as:
```gherkin
When I open the project share dialog
Then the project cannot be published
```

It would read _even_ better as a miniature user story like this, and you could ditch the comment:
```gherkin
When I am not signed in
And I open the project share dialog
Then the project cannot be published
```
In that case I think the `I am not signed in` step could just verify that the user is not signed in (which is just implied, up to this point in the test).